README.md

```diff
@@ -57,22 +57,70 @@ podman run -v $PWD:/app --platform linux/amd64 -it \
 
 ### Tests
 
+
+### Ledger VS Code extension with macOS / Windows / Linux
+
+To test your app on macOS or Windows, it is recommended to use [Ledger's VS Code extension](#with-vscode) to quickly setup a working test environment.  This extension can also 
```

Comment:
dead link, please reference this link : https://github.com/LedgerHQ/app-boilerplate#with-vscode